frontend/src/pages/Quote/index.jsx

```diff
@@ -70,7 +70,7 @@ export default function Quote() {
   const PANEL_TITLE = 'quote';
   const dataTableTitle = 'quotes Lists';
   const ADD_NEW_ENTITY = 'Add new quote';
-  const DATATABLE_TITLE = 'quotes List';
+  const DATATABLE_TITLE = 'Quotes List';
```

Comment:
thank you @karishmashuklaa for your help 
can you please change this to "Quote list"